packages/manager/apps/pci-databases-analytics/src/lib/availabilitiesHelper.ts

```diff
@@ -234,48 +239,53 @@ const mapFlavor = (
   return treeFlavor;
 };
 
+// handle plan code with or without .apac
+const getPlanCodeFromString = (planCode: string) => {
+  const hasApac = planCode.includes('.apac');
```

Comment:
Why not use the constant in packages/manager/apps/pci-databases-analytics/src/configuration/pricing.constants.ts ?

Reply:
Good catch, I missed it. Should be fixed now